pfb/commands/add.py

```diff
@@ -0,0 +1,54 @@
+import json
+import sys
+
+import click
+
+from ..base import str_hook
+from ..cli import main
+from ..reader import PFBReader
+from ..writer import PFBWriter
+
+
+@main.command(short_help="Make a blank record for add.")
+@click.argument("name")
+@click.option(
+    "-i",
+    "--input",
+    "path",
+    metavar="PFB",
+    type=click.File("rb"),
+    default="-",
+    help="Read schema from this PFB file.  [default: <stdin>]",
+)
+def make(name, path):
+    """Make a blank record according to given NODE schema in the PFB file."""
+    with PFBReader(path) as reader:
+        json.dump(reader.make_empty_record(name), sys.stdout)
+        sys.stdout.write("\n")
+
+
+@main.command(short_help="Add records into a PFB file.")
```

Comment:
What JSON input file should look like? The one from `make_empty_record`?

Reply:
Yes, that is correct, just added more docstring here.